MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/location/LocationAnimatorCoordinator.java

```diff
@@ -222,13 +222,17 @@ void feedNewTilt(double targetTilt, @NonNull CameraPosition currentCameraPositio
     playAnimators(animationDuration, ANIMATOR_TILT);
   }
 
-  private LatLng getPreviousLayerLatLng() {
+  LatLng getPreviousLayerLatLng() {
     LatLng previousLatLng;
     MapboxAnimator latLngAnimator = animatorArray.get(ANIMATOR_LAYER_LATLNG);
     if (latLngAnimator != null) {
       previousLatLng = (LatLng) latLngAnimator.getAnimatedValue();
     } else {
-      previousLatLng = new LatLng(previousLocation);
+      if (previousLocation != null) {
+        previousLatLng = new LatLng(previousLocation);
+      } else {
+        previousLatLng = new LatLng(0f, 0f);
```

Comment:
potentially this could result in a weird flickering, the location jumping from one part of the world to current location. Can we use the current position instead? This will result in a no-op animation but I feel that would be a better UX.